tools/modules/framework/manuals_markdown.sh

```
@@ -0,0 +1,220 @@
+function markdown_framework_options() {
+	mod_message="| Description | Options | Credit |\n"
+	mod_message+="|:----------- | ------- |:------:|\n"
+	# Iterate over the options
+	for key in "${!framework_options[@]}"; do
+		# Split the key into function_name and type
+		IFS=',' read -r function_name type <<< "$key"
+		# If the type is 'feature', append the option to the help message
+		if [[ "$type" == "feature" ]]; then
+			ref_link=${framework_options["$function_name,ref_link"]}
+			doc_link=${framework_options["$function_name,doc_link"]}
+			ref_link_md=$([[ -n "$ref_link" ]] && echo "[Source]($ref_link)" || echo "X")
```

Shellcheck (0.10.0)

[warning] ref_link_md appears unused. Verify use (or export if used externally).

(SC2034)